adal/oauth2_client.py

```diff
@@ -166,10 +167,8 @@ def _validate_token_response(self, body):
         try:
             wire_response = json.loads(body)
         except ValueError:
-            self._log.info(
-                'The token response from the server is unparseable as JSON: %s',
-                body)
-            raise 
+            self._log.exception('The token response from the server is unparseable as JSON')

```

Comment:
I think we should still log the `body`, unless python's `logging.exception` will do it

Reply:
Adjusted in latest commit.